src/features/ShowHelp.ts

```diff
@@ -22,7 +22,7 @@ export class ShowHelpFeature implements IFeature {
                     "Unable to instantiate; language client undefined.");
                 return;
             }
-            if (item === undefined) {
+            if (item === undefined || !item.hasOwnProperty("Name")) {
```

Comment:
Do we need to consider the scenario where the Name property exists but its value is undefined?

Reply:
yeah, that will work better. I think when I was testing it out in Chrome console it was erroring out when Name wasn't there. However, I've changed it to just test for undefined and it doesn't seem to have an issue. I'll get this changed to just test if it's undefined.

![Chrome being Chrome...](https://user-images.githubusercontent.com/30301021/52361205-8e6f6400-29f2-11e9-93e1-07de6b415898.png)

Reply:
You can just use a truthy chech:

```suggestion
            if (!item || !item.Name) {
```

Reply:
This Suggested Change feature is awesome. Being able to take that change and commit it from the browser 👌 

I also figured out why my Chrome console stuff was erroring out, I wasn't defining the variable beforehand.

Reply:
Just make sure you test the change :)

Reply:
yup, it works as expected 😃